Update LIP 0037 and LIP 0041 (#145)
LiskHQ/lips: proposals/lip-0037.md

The tag is chosen depending on the message type. Moreover, different serialization methods for the same message type require different tags. The following table defines the tags currently needed.

> | Name               | Type    | Value       | Message type                            |

Reviewer: ```suggestion
| Constant Name               | Type    | Tag          | Message type                            |
```

| ------------------ |---------| ------------| ----------------------------------------|
|`MESSAGE_TAG_TRANSACTION`  | bytes   | "LSK_TX_" as ASCII-encoded literal      | Message tag for signing transaction with serialization specified in [LIP "Define New Transaction Schema"](https://research.lisk.com/t/define-new-transaction-schema/348/15). |
|`MESSAGE_TAG_BLOCK_HEADER` | bytes   | "LSK_BH_" as ASCII-encoded literal      | Message tag for signing block headers with serialization specified in [LIP 0055](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0055.md#unsigned-block-header-json-schema). |

> |`MESSAGE_TAG_MULTISIG_REG` | bytes   | "LSK_RMSG_" as ASCII-encoded literal    | Message tag for the signatures contained in the [register multisignature group command](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0041.md#RegisterMultisignatureGroup). |

Reviewer: ```suggestion
|`MESSAGE_TAG_MULTISIG_REG` | bytes   | "LSK_RMSG_" as ASCII-encoded literal    | Message signed by the signatures contained in the [register multisignature group command](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0041.md#RegisterMultisignatureGroup). |
```

> | `MESSAGE_TAG_CHAIN_REG_MESSAGE` | bytes  | "LSK_CHAIN_REGISTRATION_" as ASCII-encoded literal | Message tag for the signatures contained in the [chain registration message](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0043.md). |

Reviewer: ```suggestion
| `MESSAGE_TAG_CHAIN_REG_MESSAGE` | bytes  | "LSK_CHAIN_REGISTRATION_" as ASCII-encoded literal | Message signed by signatures contained in the [chain registration message](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0043.md). |
```

| `MESSAGE_TAG_POA`         | bytes   | "LSK_POA_" as ASCII-encoded literal | Message tag for the signatures contained in the [update authority command](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0047.md#authority-registration-command). |

> | `MESSAGE_TAG_CERTIFICATE `| bytes  | "LSK_CE_" as ASCII-encoded literal | Message tag for signing certficates with serialization specified in [LIP 0061](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0061.md#signcertificate). |

Reviewer: ```suggestion
| `MESSAGE_TAG_CERTIFICATE `| bytes  | "LSK_CE_" as ASCII-encoded literal | Certficates with serialization specified in [LIP 0061](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0061.md#schema). |
```
